src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int [] values = readFromfile(fromFileName);
+        String report = createReport(values);
+        writeToFile(toFileName, report);
+
+    }
+
+    private int[] readFromfile(String fileName) {
```

Comment:
you are not just reading file content here, you  are processing it, I would move that part to a method dedicated to generating report